wrappers/winuser.go

```diff
@@ -206,14 +206,17 @@ const (
 	EDD_GET_DEVICE_INTERFACE_NAME = 0x00000001
 )
 
+type MonitorEnumProc func(hmonitor HMONITOR, hdc HDC, rect *RECT, lparam LPARAM) uintptr
+
 var (
 	moduser32 = syscall.NewLazyDLL("user32.dll")
 
-	procCloseDesktop       = moduser32.NewProc("CloseDesktop")
-	procEnumDisplayDevices = moduser32.NewProc("EnumDisplayDevicesW")
-	procGetSystemMetrics   = moduser32.NewProc("GetSystemMetrics")
-	procOpenInputDesktop   = moduser32.NewProc("OpenInputDesktop")
-	procSetThreadDesktop   = moduser32.NewProc("SetThreadDesktop")
+	procCloseDesktop        = moduser32.NewProc("CloseDesktop")
+	procEnumDisplayDevices  = moduser32.NewProc("EnumDisplayDevicesW")
```

Comment:
Also, looks like we're missing a wrapper for `EnumDisplayDevices`, even though it's used in `gowin32.EnumAllDisplayDevices`.  Did you test this all before submitting?

Reply:
I do not understand. There was a wrapper EnumDisplayDevices in previous version. 
I tested both EnumDisplayDevices  / EnumDisplayMonitors and they work good.

Reply:
You're right.  My mistake here.